CHANGELOG.md

```diff
@@ -3,22 +3,23 @@
 
 ## unreleased
 * PaymentButtons
-    * Deprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
-    * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
-
-* PayPalWebPayments
-  * Add `vault(url:)` method to `PayPalWebCheckoutClient`
-  * Add `PayPalVaultResult` type to return vault result
-  * Add `PayPalVaultDelegate` to handle results from vault flow
-  * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
-* PaymentButtons
+  * Deprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
+  * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
   * Add `custom` case for `PaymentButtonEdges`
   * Support VoiceOver by adding button accessibility labels
+  * Add `custom` case for `PaymentButtonEdges`
+  * Support VoiceOver by adding button accessibility labels
+  * Font typeface changed to "PayPalOpen" to meet brand guidelines
 * CardPayments
   * Add `liabilityShift` property to `CardResult`
   * Add `CardClientError.threeDSVerificationError` for invalid verification
   * Add `CardClientError.missingDeeplinkURLError` for missing deeplink URL
   * Add `CardClientError.malformedDeeplinkURLError` for malformed or invalid deeplink
+* PayPalWebPayments
```

Comment:
Do we need to merge the latest from `main` in? I think some of this CHANGELOG is from other PRs.

Reply:
The latest on `main` was merged in. What part of this CHANGELOG shouldn't be here?

Reply:
Ah got it, sorry I was confused by the diff but it looks like the order of things moved around a bit to keep all the button stuff together